sme_financing/main/service/fund_disbursement_service.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+
+from sqlalchemy.exc import SQLAlchemyError
+
+from sme_financing.main import db
+
+from ..models.fund_disbursement import FundDisbursement
+from ..service.funding_project_service import get_funding_project_by_id
+
+
+def update():
+    db.session.commit()
+
+
+def commit_changes(data):
+    db.session.add(data)
+    update()
+
+
+def save_disbursement(data):
+    funding_project = get_funding_project_by_id(id=data["funding_project_id"])
```

Comment:
Not sure the `data` object will have `funding_project_id` key

Reply:
Yeah